modules/microshift-nw-router-config-IPs.adoc

```diff
@@ -0,0 +1,56 @@
+// Module included in the following assemblies:
+//
+// * microshift_networking/microshift-nw-router.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-config-ip-addresses_{context}"]
+= Configuring router IP addresses
+
+You can restrict the network traffic to the router by configuring specific IP addresses. For example:
+* Use cases where the router is reachable only on internal networks, but not on northbound public networks.
+* Use cases where the router is reachable only by northbound public networks, but not on internal networks.
+* Use cases where the router is reachable by both internal networks and northbound public networks, but on separate IP addresses.
+
+.Prerequisites
+
+* You have created a {microshift-short} `config.yaml` file.
+
+* The {oc-first} is installed.
+
+//Q: Any other prereqs?
```

Comment:
No other prereqs, you only need to have microshift installed and the config.yaml ready for changes.